src/util.cc

```diff
@@ -9,27 +10,30 @@ using v8::String;
 using v8::Local;
 using v8::Value;
 
-#define UTF8STRING(isolate, value, dst, length)                              \
-  do {                                                                       \
-    Local<String> string = value->ToString(isolate);                         \
-    if (string.IsEmpty())                                                    \
-      return;                                                                \
-    size_t len = StringBytes::StorageSize(isolate, string, UTF8) + 1;        \
-    if (len > sizeof(str_st_)) {                                             \
-      dst = static_cast<char*>(malloc(len));                                 \
-      CHECK_NE(dst, nullptr);                                                \
-    }                                                                        \
-    const int flags =                                                        \
-        String::NO_NULL_TERMINATION | String::REPLACE_INVALID_UTF8;          \
-    length = string->WriteUtf8(dst, len, 0, flags);                          \
-    dst[length] = '\0';                                                      \
-  } while (0);
+static int Utf8String(Isolate* isolate,
```

Comment:
Maybe call it `MakeUtf8String` or something like that.  At call sites, it looks like a constructor now.